README.md

```diff
@@ -20,12 +20,10 @@ To create a  virutal environment follow these steps:
     $ pip install pipenv
     $ git clone https://github.com/GeekyFreelancers/CollegeProject
     $ cd CollegeProject
-    $ pipenv install python==3.6.5
     $ pipenv shell
+    $ pipenv install .
```

Comment:
`pipenv .` should come before `pipenv shell`.

Reply:
Should it be `pipevn .` or `pipenv install .`?